src/main/primitives/Matrix3x3.java

```diff
@@ -0,0 +1,170 @@
+package primitives;
+
+import util.NormalCompare;
+
+/**
+ * The {@link Vector} class represents a {@link Vector} with it's base at the origin and it's head at the {@link Point}
+ * 'head'.
+ *
+ * @author Abraham Murciano
+ * @author Eli Levin
+ */
+public class Matrix3x3 {
+
+	// rows of matrix
+	public Vector r1;
+	public Vector r2;
+	public Vector r3;
```

Comment:
Better make the class immutable, as you seem to be using it everywhere as if it was immutable

```suggestion
	public final Vector r1;
	public final Vector r2;
	public final Vector r3;
```

Reply:
XD